File: src/main/java/com/dennisturco/dto/CustomerResponseDTO.java
package com.dennisturco.dto;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class CustomerResponseDTO {
    private Long id;
    private Long customerTypeId;
    private String customerTypeName;

    private String name;
    private String surname;
    private String email;
    private String phone;

    private String country;
    private String region;
    private String city;
    private String province;
    private String address;

    private String vatNumber;
    private String companyName;
    private String taxCode;

    private String landline;

    private Double lat;
    private Double lon;

    private String notes;
}

File: GestioPro.Common/DTOs/CustomerResponseDTO.cs
using GestioPro.Common.Enums;

namespace GestioPro.Common.DTOs;

public record CustomerResponseDTO(
    long Id,
    CustomerType CustomerType,
    string Name,
    string Surname,
    string Email,
    string Phone,
    string? Country,
    string? Region,
    string? City,
    string? Province,
    string? Address,
    string? VatNumber,
    string? CompanyName,
    string? TaxCode,
    string? Landline,
    double? Lat,
    double? Lon,
    string? Notes,
    DateTimeOffset InsertDate,
    DateTimeOffset LastUpdateDate,
    bool IsDisabled,
    int QuotationCount,
    int ContractCount
);
